src/testRunner/runner.ts

```diff
@@ -66,6 +67,8 @@ export function createRunner(kind: TestRunnerKind): RunnerBase {
             return new FourSlashRunner(FourSlash.FourSlashTestType.Server);
         case "project":
             return new project.ProjectRunner();
+        case "transpile":
+            return new TranspileRunner();

```

Comment:
this is a mega good idea

Reply:
When we have time, we should port the bulk of the existing `transpile` unit tests to this runner, probably. (The ones that don't pass in a custom transformer, at least.)

I also have a backlogged idea to make a runner for `tscWatch`-type tests, because the incredible boilerplate for those gets me every time I write one; just need to figure out a good way to encode the `edits` you usually want to apply in sequence into the input compiler-test-type fs description.